docker/docker-dev-direct-single-tenant/datasources/mt-graphite-api

```diff
@@ -1,7 +0,0 @@
-{
-  "name":"mt-graphite-api",
-  "type":"graphite",
-  "url":"http://localhost:8080",
-  "access":"direct",
-  "isDefault":true
-}
```

Comment:
ok that sounds good.  but I still think it's useful to have 2 datasources (one to talk to graphite, and one to talk to MT). maybe not in the standard one, but definitely in docker-dev.